script.js

```diff
@@ -1,4 +1,24 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var results = scissorsPaperStone(input);
+  return results;
+};
+
+var scissorsPaperStone = function (input) {
+  var listOfChoice = ["scissors", "paper", "stone"];
+  var computerChoice = Math.floor(Math.random() * listOfChoice.length);
+  //var computerChoice = listOfChoice[randomChoice];
+  var userChoice = listOfChoice.indexOf(input);
+  var results = userChoice - computerChoice;
+  var winningCases = [-1, 2];
+  var losingCases = [1, -2];
+  if (winningCases.includes(results)) {
+    output = "Win";
+  }
+  if (losingCases.includes(results)) {
+    output = "Lose";
+  }
+  if (results == 0) {
+    output = "Draw";
+  }
+  return "The computer chose " + listOfChoice[computerChoice] + "the " + output;
```

Comment:
You can consider using JavaScript Template Literals `` instead of ' ' so that you can combine your string values with your variables in one sentence without repeated plus signs. This will save you a lot of time.

https://www.w3schools.com/js/js_string_templates.asp